int.go

```diff
@@ -503,5 +504,9 @@ func (z *intLogger) StandardLogger(opts *StandardLoggerOptions) *log.Logger {
 		opts = &StandardLoggerOptions{}
 	}
 
-	return log.New(&stdlogAdapter{z, opts.InferLevels}, "", 0)
+	return log.New(z.StandardWriter(opts), "", 0)
+}
+
```

Comment:
Let's put a comment here to indicate what this method is for, so it shows up in godoc.

Reply:
This is the internal logger. The interface has a doc comment.

Reply:
aaaaaaaah yes.